rskj-core/src/test/resources/test-rskj.conf

```diff
@@ -161,6 +161,7 @@ vm.structured.storage.dictionary.enabled = false
 
 miner.client.enabled = false
 miner.server.enabled = false
+miner.gasPriceBuffer = 105
```

Comment:
Before this change the expected value would be 110. Having this new property here, wouldn't affect other tests using previous default values?

Reply:
This value is used only by [RskSystemPropertiesTest](https://github.com/rsksmart/rskj/pull/2220/files#diff-f6561bc832eef1ae925e5bd576c6e88f42c7e6919f60d8f805fbb11b0ca09162R231) just to test teh new method added to that class so it should not affect other tests.